src/components/UserProfile.css

```diff
@@ -0,0 +1,60 @@
+.profileContainer {
```

Comment:
Consider using kebab-case for CSS selectors as it is a common convention
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/